docs/book/src/reference/using_an_external_type.md

```diff
@@ -0,0 +1,145 @@
+# Using an External Type
+
+There are several different external types that may be referenced when writing a controller.
+* A Custom Resource Definition (CRD) that is defined in the current project `kubebuilder create api`.
+* A Core Kubernetes Resources eg. `kubebuilder create api --group apps --version v1 --kind Deployment`.
```

Comment:
I think this *could* be confusing for those that are new to writing controllers and make them think that in order to use a core Kubernetes resource they will need to run `kubebuilder create api ...`. 

It may be nice to mention or somehow point to the fact that most of the core Kubernetes resources can be imported.

```suggestion
* A Core Kubernetes Resources.
```